.github/workflows/stale_issue_bot.yml

```diff
@@ -0,0 +1,22 @@
+name: Close Stale Issues
+description: "Close issues with no recent activity"
+on:
+  workflow_dispatch:
+    inputs:
+      staleDays:
+        description: "Days before an issue being marked as stale and closed"
+        required: true
+
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          close-issue-message: 'To clean up and re-prioritize bugs and feature requests we are closing all issues older than 6 months as of March 1, 2023. If there are any issues or feature requests that you would like us to address, please re-create them. For urgent issues, opening a support case with this link Snowflake Community is the fastest way to get a response'
```

Comment:
Do we specify a date when we run this action?

Reply:
We could only specify days, but not a date :/ There is a date but that is used to ignore issues before a certain date :(

https://github.com/actions/stale#days-before-stale